Library/ViewModels/RootViewModel.swift

```diff
@@ -500,13 +500,13 @@ private func tabData(forUser user: User?) -> TabBarItemsData {
     (user?.stats.memberProjectsCount ?? 0) > 0
   let items: [TabBarItem]
 
-  switch (featureCreatorDashboardEnabled(), isMember) {
-  case (false, _), (true, false):
+  switch isMember {
```

Comment:
So the idea is that the tabs are based on the user. And they don't need gating because this tab data is only used for styling - the actual setting of the tabs in the tab bar is done in `setViewControllers`.